src/AVFoundation/Enums.cs

```diff
@@ -232,6 +232,8 @@ public enum AVError : long {
 		NoCompatibleAlternatesForExternalDisplay = -11868,
 		[NoWatch, iOS (11,2), TV (11,2), Mac (10,13,2)]
 		NoSourceTrack = -11869,
+		[NoWatch, NoiOS, NoTV, Mac (11,0)]
+		RosettaNotInstalled = -11877,
```

Comment:
minor: we don't normally include availability on error codes

filed as an intro enhancement -> https://github.com/xamarin/xamarin-macios/issues/9724